astroquery/alma/tests/test_alma_remote.py

```diff
@@ -35,7 +35,7 @@ def get_client():
     # mechanism is supposed to be used
     from .. import core
     core.ALMA_TAP_PATH = 'obscore'
-    alma.archive_url = 'https://beta.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/'
+    alma.archive_url = 'https://alma.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/'
```

Comment:
please factor this out into a quick short PR, as a fix to the currently failing remote tests